datahub-web-react/src/app/homeV3/module/components/SmallModule.tsx

```diff
@@ -6,12 +8,29 @@ import ModuleMenu from '@app/homeV3/module/components/ModuleMenu';
 import { ModuleProps } from '@app/homeV3/module/types';
 import { FloatingRightHeaderSection } from '@app/homeV3/styledComponents';
 
-const Container = styled.div`
+const DragIcon = styled(Icon)<{ isDragging: boolean }>`
+    cursor: ${(props) => (props.isDragging ? 'grabbing' : 'grab')};
+    display: none;
+    position: absolute;
+    left: 4px;
+    top: 50%;
+    transform: translateY(-50%);
+`;
+
+const ContainerWithHover = styled.div`
     display: flex;
     flex-direction: column;
     position: relative;
     height: 100%;
     justify-content: center;
+
+    :hover {
+        background: linear-gradient(180deg, #fff 0%, #fafafb 100%);
+    }
+
+    :hover ${DragIcon} {
+        display: block;
+    }
```

Comment:
can we move this icon over a little here? this is what it looks like in the UI now how it seems to bump right next to the icon:
<img width="225" height="114" alt="image" src="https://github.com/user-attachments/assets/ca063809-1634-44ee-bf45-e4411b531b02" />

while in the designs there's some more room and it's bumped to the left a bit:
<img width="457" height="275" alt="image" src="https://github.com/user-attachments/assets/bdbf05d2-5ecc-4819-9953-85ecf0403555" />